fetch-logs/action.yml

```diff
@@ -0,0 +1,53 @@
+name: fetch logs
+description: fetch network logs
+inputs:
+  provider:
+    description: The cloud provider. Accepts 'aws' or 'digital-ocean'.
+    required: true
+  retry_attempts:
+    description: retry attempts
+    required: true
+  rust_log:
+    description: log level to obtain
+    required: true
+  testnet_name:
```

Comment:
Can you please change this to `network-name`? We started using `network-name` rather than `testnet-name` for the other actions.